### PR TITLE
Write Kafka docs

### PR DESCRIPTION
**Pull request by Sternakt**

### CODE REVIEW

File: .faststream_gen/kafka/message.md [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
 
-## Message access
+## Message Access
 
-You can get it in a simple way too: just acces to the message object in the [Context](../getting-started/context/existed.md){.internal-link}!
+You can easily access this information by referring to the message object in the [Context](../getting-started/context/existed.md)!
+
+This object serves as a unified **FastStream** wrapper around the native broker library message (for example, `aiokafka.ConsumerRecord` in the case of *Kafka*). It contains most of the required information, including:
 
-It is an unified **FastStream** wrapper around native broker library message (`aiokafka.ConsumerRecord` in the *Kafka* case). It contains most part of required information like:
```
Comment (davorrunje) [inline .faststream_gen/kafka/message.md:9]: why are parts about aiokafka.ConsumerRecord removed?

File: .faststream_gen/kafka/message.md [hunk 2]
```diff
@@ -24,80 +32,23 @@ async def base_handler(
...
-    raw: ConsumerRecord = msg.raw_message
-    print(raw)
+    print(msg.headers)
 ```
 
 ## Message Fields access
 
-But in the most cases you don't need all message fields, you need to know just one of them. You can use [Context Fields access](../getting-started/context/fields.md){.internal-link} feature for this reason.
-
-Like an example, you can get access to the `correlation_id` like this:
```
Comment (davorrunje) [inline .faststream_gen/kafka/message.md:46]: why is this removed?

File: .faststream_gen/kafka/message.md [hunk 3]
```diff
@@ -24,80 +32,23 @@ async def base_handler(
...
 @broker.subscriber("test")
 async def base_handler(
     body: str,
-    key: bytes | None = Context("message.raw_message.key"),
+    headers: str = Context("message.headers"),
 ):
-    print(key)
+    print(headers)
 ```
-
```
Comment (davorrunje) [inline .faststream_gen/kafka/message.md:71]: why is this removed?

**Review by davorrunje — APPROVED** [review]

**Comment by Lancetnik** [comment]
Let me fix some styles

**Review by Lancetnik — APPROVED** [review]
